packages/terra-pills/tests/wdio/pills-spec.js

```diff
@@ -0,0 +1,118 @@
+const ignoredA11y = {
+  'aria-roles': { enabled: false },
```

Comment:
Why ignore this in a spec?

Reply:
Changed cfb6aad5b38a9957f1449aa2f75bbf5d001c3c96